numpy/hook-numpy.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+# --- Copyright Disclaimer ---
+#
+# In order to support PyInstaller with numpy<1.20.0 this file will be
+# duplicated for a short period inside PyInstaller's repository [1]. However
+# this file is the intellectual property of the NumPy team and is under the
```

Comment:
This is fine to say if you prefer, but not necessary - you wrote this, so copyright remains yours.

Reply:
PyInstaller is GPL and GPL mandates every source file has a copyright header so our copy will say that whether we like it to or not but I don't see why your copy should need it to so I think I'll remove it.